install_config/install/prerequisites.adoc

```diff
@@ -362,6 +362,124 @@ SELINUX=enforcing
 SELINUXTYPE=targeted
 ----
 
+[[prereq-cloud-provider-considerations]]
+
+=== Cloud Provider Considerations
+
+*Set up the Security Group*
+
+When installing on AWS or OpenStack, ensure that you set up the security group.
+
+[cols="1,2"]
+|===
+|*All OpenShift Hosts*
+a|tcp/22 from host running the installer/Ansible
+
+|*etcd Security Group*
+a| - tcp/2379 from Masters
+- tcp/2380 from etcd hosts
+
+|*Master Security Group*
+a| - tcp/8443 from 0.0.0.0/0
+- tcp/53 from All OpenShift hosts
+- udp/53 from All OpenShift hosts
+
+|*Node Security Group*
+a| - tcp/10250 from Masters
+- tcp/4789 from Nodes
+
+|*Infrastructure Nodes*
+(ones that can host the `*openshift-router*`)
+a|- tcp/443 from 0.0.0.0/0
+- tcp/80 from 0.0.0.0/0
+
+|===
+
+If configuring ELBs for load balancing the Masters and/or routers, you also need
+to configure Ingress and Egress security groups for the ELBs appropriately.
+
+*Override Detected IP Addresses and Host Names*
+
+Some deployments require that the user override the detected host names and IP
+addresses for the hosts. To see the default values, run the `*openshift_facts*`
+playbook:
+
+====
+----
+# ansible-playbook playbooks/byo/openshift_facts.yml
+----
+====
+
+Now, verify the detected common settings. If they are not what you expect them
+to be, you can override them.
```

Comment:
I think this would be a good spot to mention that all (most) of the available Ansible variables are listed in greater detail in the Advanced Installation topic.